src/data/row.rs

```diff
@@ -15,8 +15,8 @@ pub enum RowError {
     #[error("lack of required column: {0}")]
     LackOfRequiredColumn(String),
 
-    #[error("literals does not fit to columns")]
-    LackOfRequiredValue(String),
+    #[error("do not match columns and values")]
+    DoNotMatchColumnsAndValues,
```

Comment:
Error name or message is for describing what's wrong happened. so, imperative sentence might not be a good choice.
How about simply `ColumnAndValuesNotMatched`?

Reply:
That's a good comment. changed it